dcs_core/core/validation/completeness_validation.py

```diff
@@ -89,3 +89,47 @@ def _generate_metric_value(self, **kwargs) -> Union[float, int]:
             )
         else:
             raise ValueError("Invalid data source type")
+
+
+class CountEmptyStringValidation(Validation):
+    def _generate_metric_value(self, **kwargs) -> Union[float, int]:
+        if isinstance(self.data_source, SQLDataSource):
+            return self.data_source.query_get_empty_string_count(
+                table=self.dataset_name,
+                field=self.field_name,
+                filters=self.where_filter if self.where_filter is not None else None,
+            )
+        elif isinstance(self.data_source, SearchIndexDataSource):
+            return self.data_source.query_get_empty_string_count(
```

Comment:
This has not been implemented for search data source. We should use `notImplementedError`